### DISCUSSION TITLE
Error when cloning the repo

### DISCUSSION THREAD

**Discussion by KibiDabi** [openstatusHQ/openstatus#785] [Q&A]
When I execute git clone I get this: 

Cloning into 'openstatus'...
remote: Enumerating objects: 14991, done.
remote: Counting objects: 100% (14991/14991), done.
remote: Compressing objects: 100% (5700/5700), done.
remote: Total 14991 (delta 9327), reused 13381 (delta 8417), pack-reused 0
Receiving objects: 100% (14991/14991), 14.66 MiB | 9.31 MiB/s, done.
Resolving deltas: 100% (9327/9327), done.
error: invalid path 'apps/docs/api-reference/incident/get-incident:id.mdx'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

After that I tried to run pnpm install and I get this message: 

 ERR_PNPM_NO_PKG_MANIFEST  No package.json found in C:\Users\sljevar\My\React_Projects\Cloned_Projects\openstatus

Can you please help me with this?

**Comment by thibaultleouay** [comment]
Hello , I have tried on osX and it just worked 

![CleanShot 2024-04-25 at 19 41 25](https://github.com/openstatusHQ/openstatus/assets/13894054/1f75336a-6ff0-4079-88f5-e08965f4a0f1)


It is because of the colon in the file `get-incident:id.mdx` 
Which are forbidden on windows Im going to do something for that

**Reply by KibiDabi** [reply]
Great. Please let me know when you solve it. Thank you very much!

**Comment by thibaultleouay** [comment]
@KibiDabi  it has been solved
